src/components/WhoWeAre/index.css

```diff
@@ -0,0 +1,58 @@
+@import url("https://fonts.googleapis.com/css?family=Acme&display=swap");
+
+.whoWeAre {
+  font-family: "Acme";
+}
+.targetText {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 1.8em;
+  margin: 1.8em;
+  width: 100%;
+  line-height: 170%;
+  text-align: center;
+  height: auto;
+}
+
+.targetHeader {
+  font-size: 1.8em;
+  color: #03c4a1;
+}
+
+.targetParagraph {
+  padding: 1.8em;
+  font-size: 1.1em;
+  color: #726a95;
+}
+
+img {
```

Comment:
Can you give this img a className, because it affects all the imgs in the websites this way